source/Minable.cpp

```diff
@@ -133,7 +135,7 @@ void Minable::Place(double energy, double beltRadius)
 	// apoapsis distance is no closer than .8: scale >= .8 * (1 - e)
 	double sMin = max(.4 * (1. + eccentricity), .8 * (1. - eccentricity));
 	double sMax = min(4. * (1. - eccentricity), 1.3 * (1. + eccentricity));
-	orbitScale = (sMin + Random::Real() * (sMax - sMin)) * beltRadius;
+	orbitScale = (sMin + spread * Random::Real() * (sMax - sMin)) * beltRadius;
```

Comment:
Same here. Have the comments explain why spread is being used here and the effect that this spread multiplication has on the position of a set of asteroids.